taskcluster/configs/config.prod.yml

```diff
@@ -161,6 +161,8 @@ taskcluster:
   # After the parallel corpora are merged and de-duplicated, the combined file is
   # then split into an even number of chunks.
   split-chunks: 10
+  worker-classes:
+    default: gcp-spot
```

Comment:
Suggestion: It would be helpful to document the available values here, or point to where they are defined (taskcluster/translations_taskgraph/actions/train.py)

Reply:
That's a good idea; will do.